yarn.lock

```diff
@@ -783,6 +813,14 @@ babel-preset-jest@20.1.0-delta.1:
   dependencies:
     babel-plugin-jest-hoist "20.1.0-delta.1"
 
+babel-preset-minify@0.0.0:
+  version "0.0.0"
```

Comment:
Should we revert changes to Yarn lock? Seems like Uglify update could also go into a separate PR with just necessary lockfile changes.